package.json

```diff
@@ -1,25 +1,57 @@
 {
-  "name": "jbrowse-next",
+  "name": "@overture/dms-jbrowse",
   "version": "0.1.0",
-  "private": true,
+  "description": "Jbrowse components for DMS",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/overture-stack/dms-jbrowse-components"
+  },
   "scripts": {
-    "dev": "next dev",
-    "build": "next build",
-    "start": "next start",
-    "lint": "next lint"
+    "build::assets": "cp ./package.json ./dist/package.json && cp ./README.md ./dist/README.md",
```

Comment:
again based on argo uikit build config

Reply:
this is only copying package.json & readme.md, but I renamed it to `build::config` for clarity. at the moment we do want to publish the whole dist folder - all the nextjs extras have been removed